matrix-sdk/src/main/java/org/matrix/androidsdk/rest/api/FilterApi.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2015 OpenMarket Ltd
```

Comment:
Replace by Copyright 2108 (New Vector Ltd) please

Reply:
Removed the wrong line while Copy/Pasting...